src/backend/base/langflow/components/tools/astradb.py

```diff
@@ -134,6 +238,11 @@ def projection_args(self, input_str: str) -> dict:
         elements = input_str.split(",")
         result = {}
 
+        if elements == ["*"]:
+            return None
+
+        result["$vector"] = False
```

Comment:
could we add a comment here on explaining what this is doing? is it ignoring the `$vector` field in the output of the tool component?